File: Angular-app/src/app/pages/classes/station.ts
import { Line } from './line';

export class Station {
    Id: number;
    Name: string;
    Address: string;
    X: number;
    Y: number;
    Lines: Line[] = [];

    constructor(station: Station) {
        this.Id = station.Id;
        this.Address = station.Address;
        this.Name = station.Name;
        this.X = station.X;
        this.Y = station.Y;
        this.Lines = station.Lines;
    }
}

File: WebApp/Models/Station.cs
﻿using System.Collections.Generic;

namespace WebApp.Models
{
    public class Station
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Address { get; set; }

        public double X { get; set; }

        public double Y { get; set; }

        public List<Line> Lines { get; set; }
    }
}